source-control/readme.md

```diff
@@ -3,17 +3,25 @@
 There are many different options when working with Source Control. In [CSE](../CSE.md) we use [AzureDevOps](https://csesd.visualstudio.com/_projects) for private repositories and [GitHub](https://github.com/) for public repositories.
 
 ## Sections within Source Control
+
 * [Git](git.md)
 * [Feature Branching](feature-branching/readme.md)
 * [Versioning](versioning/readme.md)
 
 ## Goal
+
 * Following industry best practice to work in geo-distributed teams which encourage contributions from all across [CSE](../CSE.md) as well as the broader OSS community
 * Improve code quality by enforcing reviews before merging into master branches
 * Improve traceability of features and fixes through a clean commit history
 
 ## General Guidance
-Consistency is important, so agree to the approach as a team before starting to code. The team should at least be doing the following:
+
+Consistency is important, so agree to the approach as a team before starting to code. Consider this as a design decision
```

Comment:
Some re-wording suggestions to make it clearer and more concise "Consistency is important, so agree on the approach as a team before starting to code. Treat this as a design decision, so include a design proposal and review, in the same way as you would document all design decisions (see [Working Agreements](../team-agreements/working-agreements/readme.md))."

Reply:
Updated.